full-stack-tests/core/src/frontend/standalone/PerModelCategoryVisibility.test.ts

```diff
@@ -311,7 +311,7 @@ describe("Per-model category visibility overrides with setOverrides function", (
     overrides.push({modelId: "0x1f", categoryIds: "0x17", visOverride: hide});
     await pmcv.setOverrides(overrides);
 
-    expect(pmcv.getOverride("0x1c", "0x2f")).to.equal(show);
+    expect(pmcv.getOverride("0x1c", "0x2f")).to.equal(show); // These are false. We get expected 0 to equal 1.
```

Comment:
I don't understand this comment.

Reply:
Sorry.. good catch. Thats not supposed to be there